modules/qna/src/backend/api.ts

```diff
@@ -12,10 +12,15 @@ export default async (bp: typeof sdk, bots: ScopedBots) => {
     try {
       const { storage } = bots[req.params.botId]
       const items = await storage.fetchItems(req.params.topicName)
-      // TODO: implement filtering
-      const data = { count: items.length, items }
+      const filteredItems = items.filter(qna =>
+        // @ts-ignore
+        [...Object.values(qna.questions).flat(), ...Object.values(qna.answers).flat()].filter(q => q.includes(req.query.question)).length > 0
```

Comment:
can you compare in a non-case sensitive way?  ie. `.toLowerCase()` on both strings